gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GaaSFlowObservabilityEvent.avsc

```diff
@@ -0,0 +1,82 @@
+{
+  "type": "record",
+  "name": "GaaSFlowObservabilityEvent",
+  "namespace": "org.apache.gobblin.metrics",
+  "doc": "An event schema for GaaS to emit during and after a flow is executed.",
```

Comment:
please clarify under what circumstances we anticipate emitting the event *during* flow execution

Reply:
Was thinking eventually this would happen in the future, but for now we will keep events to only emit on completion to match current behavior